Index: b/debian/rules
===================================================================
--- a/debian/rules
+++ b/debian/rules
@@ -10,6 +10,9 @@ include /usr/share/dpkg/default.mk
 # see FEATURE AREAS in dpkg-buildflags(1)
 export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
+# improve compatibility with newer gcc versions
+export DEB_CFLAGS_MAINT_APPEND = -Wno-address -Wno-stringop-truncation
+
 # main packaging script based on dh7 syntax
 %:
 	dh $@ --with autotools-dev
